packages/gatsby-theme-docs/src/utils/helpers/compute-path-with-path-prefix.js

```diff
@@ -0,0 +1,8 @@
+import { withPrefix } from 'gatsby-link';
+
+export default function computePathWithPathPrefix(path) {
```

Comment:
I wondering if we really need this, I would expect to just do:

```js
withPrefix(path)
```

Then there is no reason for this helper function. Can you maybe explain what problems is this trying to solve and where is it supposed to be used?

Reply:
Each microsite was generated with path prefixes, e.g., for our test site the url would be:

`http://docs.commercetools.com/test`

Writers would write a path, say to getting started like so:

`/getting-started/`

Although GatsbyLink should generate the path prefix it did not work for some reason so we ended up with a link like `http://docs.commercetools.com/getting-started/`

Hence this function.

Reply:
Sure, but the signature of the `withPrefix` method accepts a path

<img width="950" alt="image" src="https://user-images.githubusercontent.com/1110551/68387826-8bf79180-015f-11ea-9eb8-a0b96e8ff41d.png">

This custom helper function is basically doing the same thing, that's why I'm asking why do we need it in the first place and not just use `withPrefix(path)` 🤔